### DISCUSSION TITLE
Django Rest Framework website is not working

### DISCUSSION THREAD

**Discussion by Olamidun** [encode/django-rest-framework#8023] [General]
Hey guys,
![Screenshot 2021-06-06 231925](https://user-images.githubusercontent.com/59403223/120941999-cb140d00-c71d-11eb-8772-2ba50f7c1c32.png)

[https://www.django-rest-framework.org/](https://www.django-rest-framework.org/) does not seem to be working, I have been trying it since morning, does anybody know if the team is aware and working on it?

**Comment by lovelydinosaur** [comment]
Looks fine from here.

**Reply by Olamidun** [reply]
Hey Tom,
Thanks for the response and thank you for the great work you do with Django rest framework, it's very much appreciated. 

I still can't seem to access it,  I'm sure it's not my internet connection because I can access other websites,  any idea what I gotta do?

**Reply by lovelydinosaur** [reply]
🤷‍♂️ Not sure, I'd start by checking if you're able to access it from another device?

**Comment by donaldte** [comment]
Hey guys!! it's like djangorestframework webiste is not opening anymore . ?????

**Reply by lovelydinosaur** [reply]
It's working fine.

**Reply by donaldte** [reply]
alright thanks i just try again while using a vpn and is opening now . Thanks
